src/core/reactor.cc

```diff
@@ -2916,7 +2917,8 @@ void reactor::start_aio_eventfd_loop() {
     future<> loop_done = repeat([this] {
         return _aio_eventfd->readable().then([this] {
             char garbage[8];
-            ::read(_aio_eventfd->get_fd(), garbage, 8); // totally uninteresting
+            auto res = ::read(_aio_eventfd->get_fd(), garbage, 8); // totally uninteresting
+            (void)res;
```

Comment:
nit, how about using `std::ignore`? i imagine if we can replace all of them with `_` in C++26. as `std::ignore` is easier to grep than `void` , and visually more obvious than the later. so i prefer `std::ignore`.

Reply:
Thanks of the note, changed to `std::ignore`.